lutris/sysoptions.py

```diff
@@ -99,77 +133,45 @@ def get_vk_icd_choices():
     amdvlk = []
     amdvlkpro = []
     icd_files = defaultdict(list)
+
+    vulkan_data_dirs = set()
+    for env_var in VULKAN_DATA_DIR_ENV_VARS:
```

Comment:
This logic does not entirely agree with [your link](https://github.com/KhronosGroup/Vulkan-Loader/blob/v1.3.235/docs/LoaderDriverInterface.md#driver-discovery-on-linux). That document describes how it falls back to other things when those environment variables are not available.

I think your logic falls back even if they are, I think.

Reply:
That's a good point, I'll try to make the search order match that more exactly.

Reply:
Hmm, not sure the ordering matters here: the order matters for the vulkan loader selecting a default device.
It doesn't matter here because we just want to show all available ICDs for the user to pick between.

What about adding the extra fallback options of $HOME/.config and /etc/xdg and /etc and /usr/share that are currently missing but not trying to do anything about the ordering?